pkg/controllers/hub/trafficmanagerbackend/controller.go

```diff
@@ -523,6 +523,9 @@ func isValidTrafficManagerEndpoint(export *fleetnetv1alpha1.InternalServiceExpor
 	if export.Spec.IsInternalLoadBalancer {
 		return fmt.Errorf("internal load balancer is not supported")
 	}
+	if export.Spec.PublicIPResourceID == nil {
```

Comment:
do we not check the format of the resources? it seems like "abc" used throughout the tests is probably not a valid value?

Reply:
we don't check the format, cause it should read from the arm response, assuming it should always be valid.